### PR TITLE
Swing UI and other stuff

### PR DESCRIPTION
**Pull request by ThatZeoGal**
- new chart markers default to "" instead of "New Section"
- swing UI goes from 50% to 75%
- swing UI has buttons to select swing presets (Straight, Light, Standard, Heavy)
- new "documentation" for how swing is calculated and stored

### CODE REVIEW

**Comment by ThatZeoGal** [comment]
once again, please tag this for the "minor" branch or whatever it's called

**Comment by fancythedeveloper** [comment]
> * swing UI goes from 50% to 75%

would this mean you can't put a swing ratio above 75%?

**Comment by ThatZeoGal** [comment]
currently yes, however this is actually the same as it was before; it was just labelled as 100% for some reason. no functionality or limits were changed :)

File: Assets/Scripts/LevelEditor/Timeline/SpecialTmeline/TempoDialog.cs [hunk 1]
```diff
@@ -97,24 +97,81 @@ public void HalveTempo()
...
             float swing = float.Parse(swingInput.text);
-            tempoObj.SetSwing(swing * 0.25f / 100f);
-            swingInput.text = (tempoObj.chartEntity["swing"] * 400).ToString("F");
-            swingSlider.value = tempoObj.chartEntity["swing"];
+            tempoObj.SetSwing(swing / 100f - 0.5f);
+            RefreshSwingUI();
+        }
+    }
+
+    /*
```
Comment (minenice55) [inline Assets/Scripts/LevelEditor/Timeline/SpecialTmeline/TempoDialog.cs:125]: to replace with proper xml doc later

**Pull request merged**